flow.go

```diff
@@ -278,12 +276,27 @@ func Use(middlewares ...Middleware) {
 func (f *Flow) Use(middlewares ...Middleware) {
 	for _, m := range middlewares {
 		if m == nil {
-			panic("middleware cannot be nil")
+			panic("middlewares cannot be nil")
```

Comment:
Why using plural? You are only checking one middle ware at a time

Reply:
Thanks https://github.com/goyek/goyek/pull/419